etc/restic/_global.env.sh

```diff
@@ -30,6 +30,8 @@ export RESTIC_BACKUP_EXTRA_ARGS=
 # Override this value in a profile if needed.
 export RESTIC_VERBOSITY_LEVEL=0
 
-# (optional) Desktop notifications. See restic_backup.sh for details on how to set this up.
-export RESTIC_NOTIFY_BACKUP_STATS=false
+# Backup summary stats log: snapshot size, etc. (empty/unset won't log)
+export RESTIC_BACKUP_STATS_DIR="$INSTALL_PREFIX/var/log/restic-automatic-backup-scheduler"
```

Comment:
Could we make this opt-in please, to make it as simple as possible for new users. Advanced users can enable it. We can include an out-commented line as an example of how to enable it.

Reply:
Hi! sure!! But just want to double-check it :)

Do you think it is better to have the log disabled by default? Probably I'm biased and looking from my use-case perspective only, but I guessed that would be a sane default to have a summary log enabled by default for all users, and make it an opt-out. For myself I find to log this info an essential feature.

Maybe looking from my limited use case here, hehe (I'm just doing a daily backup of home). The only **cons** I see of having it enabled by default is that it needs to run `restic snapshots` and `restic diff` (I stated it on the README 🙂), which takes a little more time to finish the systemd service (not the backup) and I guess requires extra talk to the remote repo (but as it runs unattended on background I don't care). The **benefit** is that you can quickly check how the backup increased on every snapshot without ceremony, and helps to investigate the backup.

Let me know, and if you still think it is better to be an opt-in I could just comment `#` this line and add the `(optional) comment` 😃, leaving the example directory as-is `/var/log/restic-auto...` but commented + update the README accordingly. Cheers!

Reply:
Hi, I see your point that it makes sense to have. I personally would like it. Though the project scope has grown a bit too much making maintenance harder (I have much less time at the moment for it). Thus I would much appreciate if this was opt-in. 🙂

Reply:
Yep, no problem! I disabled it (left it commented), you can check now https://github.com/erikw/restic-automatic-backup-scheduler/pull/128/commits/ccdf7b7a4fbd08bbc6c439ae0253669238e382cf